### PR TITLE
add disable column in spreadsheet

### PR DESCRIPTION
**Pull request by jvfdunkley**
### :pushpin: References

-   **Issue:** Closes ["Add disable column in spreadsheet"](https://app.clickup.com/t/ewz22j) in ClickUp

### :memo: Implementation 
- I added the disabled column in ImportExport
- For the export function, I just had to pass in all columns instead of filtering by the ones that are not supposed to be shown on export
- For the import I added disabled to the userCredentials field so that it would be copied to the right place when updating a user
- The disabled value would show up as a string when it had to be a boolean so I had to convert it in the getUsersFromCsv function
 
### :art: Screenshots
Screen Shot 2021-03-05 at 11 09 15 AM
Screen Shot 2021-03-05 at 11 23 48 AM

### CODE REVIEW

**Review by tokland — CHANGES_REQUESTED** [review]

File: src/models/userHelpers.js [hunk 1]
```diff
@@ -266,7 +266,6 @@ async function getUsersFromCsv(d2, file, csv, { maxUsers, orgUnitsField }) {
         : _(csv.data)
               .drop(1)
               .value();
-
     const plainUserAttributes = _(d2.models.users.modelValidations)
         .map((value, key) => (_(["TEXT", "DATE", "URL"]).includes(value.type) ? key : null))
```
[the remark this answers is not in the document]
Comment (adrianq) [reply src/models/userHelpers.js:270]: @tokland I would say we can render the boolean columns as a toggle switch. Can you take care of this @jvfdunkley?

File: src/models/userHelpers.js [hunk 2]
```diff
@@ -337,7 +336,9 @@ async function getUsersFromCsv(d2, file, csv, { maxUsers, orgUnitsField }) {
         const data = userRows.map((userRow, rowIndex) =>
             getPlainUserFromRow(userRow, modelValuesByField, rowIndex + 2)
         );
-        const users = data.map(o => o.user);
+        const users = data.map(o =>
+            o.user.disabled ? { ...o.user, disabled: o.user.disabled.toLowerCase() } : o.user
```
Comment (tokland) [inline src/models/userHelpers.js:340]: I'd keep it a boolean, something like: `disabled: o.user.disabled === "true"`

Comment (tokland) [inline src/models/userHelpers.js:340]: Sorry, I hadn't notice before, but this processing for the field disabled could be in "getPlainUserFromRow", right?

**Review by tokland — CHANGES_REQUESTED** [review]
Some in-line comments. Check with @adrianq, but I think we need some extra work on the import part. In the import table, show a toggle, and make sure a boolean value (not a string) is posted.

**Review by tokland — APPROVED** [review]
I pushed some changes for the toggle, works for me!

**Pull request merged**